index.js

```diff
@@ -75,6 +76,7 @@ class ShopifyToken {
     this.sharedSecret = options.sharedSecret;
     this.redirectUri = options.redirectUri;
     this.apiKey = options.apiKey;
+    this.proxyAgent = 'proxyAgent' in options ? options.proxyAgent : null;
```

Comment:
It can actually be any agent so I would rename the option to just `agent`.

Reply:
Updated name to `agent`